src/QDMEwald/CMakeLists.txt

```diff
@@ -1,196 +1,6 @@
-cmake_minimum_required(VERSION 3.10)
-cmake_policy(VERSION 3.10)
-project(QDMEwald)
```

Comment:
@JensWehner if you want to keep it a separate project please let me know.

Reply:
what is the difference?

Reply:
I do not care from what I read online